### ISSUE TITLE
[Feature]  A switch to turn on/off emacs-like keybindings 

### ISSUE THREAD

**Issue by LicenceETer** [typora/typora-issues#5020]
Env:
    - Version 1.0.5 (6032)
    - macOS Monterey v12.1
    - Apple M1 pro

I knew typora have added the emacs-like keybindings feature.
[#1161](https://github.com/typora/typora-issues/issues/1161)

etc:   
- ctrl-a = beginning-of-line
- ctrl-e = end-of-line
- ctrl-f = right
- ctrl-b = left
- ctrl-k = kill-to-end-of-line
- ctrl-t = transpose-two-characters
- ctrl-v = down-page

But if i  modify `ctrl + a`  to `Select All` by [***System Preferences -> Keyborad -> Shortcuts***], it still works like`beginning-of-line`.  Or if i modify `ctrl + v` to `Paste` , it will `down-page` utill cursor jumps to the bottom, and than `Paste`.

Could you please add a switch to turn on/off for `emacs-like keybindings` , so we can change those shortcuts .

**Comment by abnerlee** [comment]
> But if i modify ctrl + a to Select All by [System Preferences -> Keyborad -> Shortcuts]

Will this work in TextEdit.app ?

**Comment by LicenceETer** [comment]
Yep, it works well in TextEdit.app .

**Comment by LicenceETer** [comment]
> > But if i modify ctrl + a to Select All by [System Preferences -> Keyborad -> Shortcuts]
> 
> Will this work in TextEdit.app ?

Yep, it works well in TextEdit.app .

**Comment by droygao** [comment]
Yes, please add a switch. I have multiple OSs to work on and have redefined all shortcuts to Windows style. Ctrl+V is used very often and could not work due to the Emacs keybindings ...
